processing/src/main/java/org/apache/carbondata/processing/store/TablePage.java

```diff
@@ -104,6 +105,16 @@
     noDictDimensionPages = new ColumnPage[model.getNoDictionaryCount()];
     int tmpNumDictDimIdx = 0;
     int tmpNumNoDictDimIdx = 0;
+
+    // find columns config to build page bloom
+    List<String> pageBloomColumns = new ArrayList<>();
```

Comment:
No need to keep this new list to store bloom information. Instead in `DimensionSpec `and `MeasureSpec` add a flag for isBloomPage